app/jobs/update_viewer_sinatra_job.rb

```diff
@@ -28,32 +22,31 @@ def push_ref_is_master?
     push['ref'] == 'refs/heads/master'
   end
 
-  def files_to_update
-    to_update = {}
-    git_clone(push['repository']['full_name']) do
-      final_json.each do |file|
-        match = file.match(/^data\/(?<country>\w+)/)
-        country = match[:country]
-        sha_updated = `git log --format='%h|%at' -1 #{file}`
-        to_update[country] = sha_updated
-      end
+  def countries_json_pushed?
+    files = push['commits'].map do |commit|
+      commit['added'] + commit['modified']
     end
-    to_update
+    files = files.flatten.uniq
+    files.any? { |file| file =~ /^countries.json$/ }
```

Comment:
Hah, no there's no reason other than previously it was `file =~ /final.json$/` and I clearly made the change without giving it too much thought!